packages/ssr-web/config/environment.js

```diff
@@ -86,6 +86,9 @@ module.exports = function (environment) {
         '/^localhost:\\d+$/',
       ],
     },
+    chains: {
+      layer2: process.env.DEPLOY_TARGET === 'production' ? 'xdai' : 'sokol',
```

Comment:
DEPLOY_TARGET doesn't exist yet so this won't work.

Reply:
Should probably use `SSR_WEB_ENVIRONMENT` when https://github.com/cardstack/cardstack/pull/2725 is merged?